geminidr/core/parameters_spect.py

```diff
@@ -102,6 +102,38 @@ def validate(self):
         if self.w1 is not None and self.w2 is not None and self.w2 <= self.w1:
             raise ValueError("Ending wavelength must be greater than starting wavelength")
 
+
+class adjustSlitOffsetToReferenceConfig(config.Config):
+    suffix = config.Field("Filename suffix", str, "_slitOffsetCorrected",
```

Comment:
@chris-simpson - I used `_slitOffsetCorrected` for the suffix, unless you have a better suggestion ?